source/extensions/filters/network/mysql_proxy/mysql_codec_switch_resp.cc

```diff
@@ -0,0 +1,28 @@
+#include "extensions/filters/network/mysql_proxy/mysql_codec_switch_resp.h"
+
+#include "extensions/filters/network/mysql_proxy/mysql_codec.h"
+#include "extensions/filters/network/mysql_proxy/mysql_utils.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace MySQLProxy {
+
+void ClientSwitchResponse::setAuthPluginResp(std::string& auth_plugin_resp_) {
```

Comment:
no _ suffix for parameter, and this method does nothing as it hides member variable. so this one is not tested?